src/Microsoft.Health.Fhir.Core/Features/Search/Expressions/FieldName.cs

```diff
@@ -26,5 +26,6 @@ public enum FieldName
         TokenSystem,
         TokenText,
         Uri,
+        LastModified,
```

Comment:
Where is this used?

Reply:
I do not need it, I had added it thining it was necessary, but it can be removed.